.github/workflows/smoke-activemq.yml

```diff
@@ -36,4 +36,4 @@ jobs:
       - name: Check info Worker1
         run: (( $(curl -f -k --retry 5 --retry-connrefused --connect-timeout 30 --retry-delay 30 https://localhost:8444/fineract-provider/actuator/info | wc --chars) > 100 ))
       - name: Run Smoke Test with Remote COB
-        run: ./gradlew --no-daemon --console=plain :integration-tests:cleanTest :integration-tests:test --tests "org.apache.fineract.integrationtests.investor.externalassetowner.InitiateExternalAssetOwnerTransferTest" -PcargoDisabled
+        run: ./gradlew --no-daemon --console=plain :integration-tests:cleanTest :integration-tests:test --tests "org.apache.fineract.integrationtests.investor.externalassetowner.InitiateExternalAssetOwnerTransferTest.saleActiveLoanToExternalAssetOwnerAndBuybackADayLater" -PcargoDisabled
```

Comment:
So basically, only one of the tests to be executed?

Reply:
Yes, The goal is to run one single testcase that uses COB Remote Jobs. If that one succeeds that we can confirm that both the ActiveMQ and Kafka Integration is working well. 

The github runner is not that powerful, so running 1 postgres, 1 kafka / 1activemq, 1 fineract manager , 1 worker is already hard enough. We have seen stability problems, and reducing the number of the testcases here may also reduce the stress on the runner.